docs/tools-terminology.md

```diff
@@ -304,6 +305,20 @@ Pipeline's parameters with concrete values
 - **Trigger**: Triggers is a Kubernetes Custom Resource Defintion (CRD) controller 
 that allows you to extract information from event payloads (a "trigger") to create Kubernetes resources.
 
+### Woodpecker
+
+[Woodpecker](https://woodpecker-ci.org/) is an open source, simple CI/CD engine with great extensibility.
+
+Some of the core Tekton terms are listed below. [[22]]
+
+- **Pipeline**: Triggered by events like webhooks, crons or manual events. It contains one or more Workflows.
+- **Workflow**: Represent one task an Agent do execute. It contains steps and report the result back to SCM Tools.
```

Comment:
```suggestion
- **Workflow**: Represents a sequential list of steps which are executed as a task by an agent. The result of a workflow is reported back to the forge.
```

Reply:
I called it "SCM Tools" instead of forge as it's used to mention gitlab,github,... in this doc - not sure what is best

Reply:
```suggestion
- **Workflow**: Represents a sequential list of steps which are executed as a task by an agent. The results of the workflow reports the result back to SCM Tools.
```

Reply:
^^ Will merge after that change lands.